src/main/java/it/systems/paymentreviser/entity/CaseFactory.java

```diff
@@ -0,0 +1,4 @@
+package it.systems.paymentreviser.entity;
+
+public class CaseFactory {
```

Comment:
Is this used anywhere?

Reply:
I wanted to move functionality of creating case with switch from service into factory but after that decided that it would be an overkill. However forgot to remove this class.